sound/soc/sof/debug.c

```diff
@@ -17,6 +17,78 @@
 #include "sof-priv.h"
 #include "ops.h"
 
+static ssize_t sof_dfsentry_write(struct file *file, const char __user *buffer,
```

Comment:
Can we add a read() for this debugFS where it can return count, min time, max time and avg time.

Reply:
@lgirdwood does that mean a read would have to be followed by a write? I could modify the "write to dump the stats into dmesg after the flood test is complete. Would that be OK?

Reply:
@ranj063 
I think we should dump out what is the current test status. By show how many IPC send and how many get from the same debugfs.

Reply:
@xiulipan are you suggesting you want to maintain a history of all the times we've run this test?

Reply:
@ranj063 
So we will use demsg for all the test logs?

Reply:
@xiulipan it is the easiest method for now. If we read from the debugfs, it means I have to find a way to store the results of the last test and have a way to parse the output that the driver returns. It wont be as simple as cat ipc_test or hexdump ipc_test which will give us no info regardign the stats

Reply:
@ranj063 read() could return human-friendly ascii summary of statistics so far. I.e. like the various debug files of graphics -> /sys/kernel/debug/dri/0/  .. this cp/cat'ed out, attached to a bug filing, plus still machine parsed for automated testing (less convenient if only outputting to dmesg)